src/solver/simulation/include/antares/solver/simulation/remix-utils.h

```diff
@@ -4,48 +4,50 @@
 #include <ranges>
 #include <vector>
 
+namespace rng = std::ranges;
```

Comment:
Please don't shorten ranges as "rng". Could be confused with Random Number Generator